inc/admin/upgrader.php

```diff
@@ -356,7 +356,9 @@ function rocket_new_upgrade( $wp_rocket_version, $actual_version ) {
 		wp_safe_remote_get( esc_url( home_url() ) );
 	}
 
-	if ( version_compare( $actual_version, '3.3.2', '<' ) ) {
+	if ( version_compare( $actual_version, '3.3.2', '<' )
+		||
+		version_compare( $actual_version, '3.6.1', '<' ) ) {
```

Comment:
We don't need to keep the first `version_compare()`, since the second will be executed anyway. You can copy this block and put it at the end of the function instead